sympy/polys/tests/test_rootoftools.py

```diff
@@ -12,7 +12,7 @@
 
 from sympy import (
     S, sqrt, I, Rational, Float, Lambda, log, exp, tan, Function, Eq,
-    solve, legendre_poly
+    solve, legendre_poly, Symbol, Integral
```

Comment:
Are these extra imports needed?

Reply:
Yes and No. They are both used in test 15920, but x is already imported at the top from abc so needs not to be recreated.